splunk_add_on_ucc_framework/global_config_update.py

```diff
@@ -123,9 +123,13 @@ def _handle_xml_dashboard_update(global_config: global_config_lib.GlobalConfig)
 
 def handle_global_config_update(global_config: global_config_lib.GlobalConfig) -> None:
     """Handle changes in globalConfig file."""
-    current_schema_version = global_config.schema_version
+    current_schema_version = global_config.schema_version or "0.0.0"
     version = current_schema_version if current_schema_version else "0.0.0"
```

Comment:
If `global_config.schema_version` is not present - then we will get `None` and `version` would become "0.0.0". Why do we need to have `or "0.0.0"`?

Reply:
got the error of pre-commit: 
`Argument 1 to "_version_tuple" has incompatible type "Optional[str]"; expected "str"  [arg-type]`

Reply:
Updated the logic for the schema version update